build/common.targets

```diff
@@ -90,6 +96,95 @@
 
   <!--
     ============================================================
+    GatherSigningInputs - gathers the list of DLLs that need to
+    be signed from each project
+    ============================================================
+  -->
+  <Target Name="GetSigningInputs" DependsOnTargets="GetTargetFrameworkSet" Returns="@(DllsToSign)" Condition=" '$(Shipping)' == 'true' ">
+    <MSBuild
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="$(SignTargetsForRealSigning)"
+      Properties="TargetFramework=%(ProjectTargetFrameworkEntries.Identity);
+                  BuildProjectReferences=false;">
+
+      <Output
+          TaskParameter="TargetOutputs"
+          ItemName="DllsToSign" />
+    </MSBuild>
+    <MsBuild Condition="'$(IncludeInVsix)' == 'true' AND '$(BuildRTM)' != 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="_GetLocalizedDllsForSigning"
+      Properties="IsVsixBuild=true;
+                  BuildProjectReferences=false;">
+      <Output
+        TaskParameter="TargetOutputs"
+        ItemName="DllsToSign"/>
+    </MsBuild>
+  </Target>
+
+  <Target Name="_GetLocalizedDllsForSigning" Returns="@(LocalizedDllsToSign)">
```

Comment:
Discussed offline with Rohit, this target will pick-up all the dll included in vsix and try to localize them even if they have no resources.